client-spark/common/src/main/java/org/apache/spark/shuffle/RssSparkConfig.java

```diff
@@ -20,127 +20,264 @@
 import java.util.Set;
 
 import com.google.common.collect.Sets;
+import org.apache.spark.internal.config.ConfigBuilder;
+import org.apache.spark.internal.config.ConfigEntry;
+import org.apache.spark.internal.config.TypedConfigBuilder;
+import scala.runtime.AbstractFunction1;
 
 import org.apache.uniffle.client.util.RssClientConfig;
+import org.apache.uniffle.common.config.ConfigUtils;
 
 public class RssSparkConfig {
 
+  public static final ConfigEntry<Integer> RSS_PARTITION_NUM_PER_RANGE = createIntegerBuilder(
+      new ConfigBuilder("spark.rss.partitionNum.per.range")
+          .doc("xxxxxx"))
+      .createWithDefault(1);
```

Comment:
Could we modify this?
From
```
createWithDefault(1))
```
to
```
createWithDefault(RssClientConfig.RSS_PARTITION_NUM_PER_RANGE_DEFAULT_VALUE);
```
Because it's more easy to modify default value in the future if we need.

Reply:
done.